app/components/app-layouts/application-header.hbs

```diff
@@ -15,14 +15,36 @@
   </a>
   <div class="global-search">
 
-<PowerSelect
-  @placeholder={{t "global.form.searchPlaceholder"}}
-  @searchEnabled={{true}}
-  @search={{perform this.search}}
-  @onChange={{this.itemSelected}} as |issue|>
-  <i class="fa fa-tasks"></i> <strong>#{{issue.number}}</strong> - {{issue.name}}
-</PowerSelect>
+    <PowerSelect
+      @placeholder={{t "global.form.searchPlaceholder"}}
+      @searchEnabled={{true}}
+      @search={{perform this.search}}
+      @onChange={{this.itemSelected}} as |issue|>
+      <i class="fa fa-tasks"></i> <strong>#{{issue.number}}</strong> - {{issue.name}}
+    </PowerSelect> 
   </div>
+
+    {{!-- Create Modules e.g. Project,Issue,Conversation (START) --}}
+    <div class="btn-group create-modules" data-project-id={{this.trackedProject.id}}>
+
+      {{!-- Create Project --}}
+      
+      <AppUi::Button type="button" class="btn btn-default btn-sm" {{action this.routeToModule 'project' 'app.projects.create'}}>
```

Comment:
Will this button be visible if the logged-in user cannot create a project due to access restrictions set via RBAC?

Reply:
The components are provided with an ACL context. Now, the button will be rendered if the user has access to the resource creation.